core/src/workflows/crud_status.rs

```diff
@@ -0,0 +1,163 @@
+use crate::context::Context;
```

Comment:
The functions in this file are not workflows in the sense of the other workflows in this directory. They don’t aggregate async action creators to a high-level process. 

Actually these functions are misplaced reducers.